model/registry/file.yaml

```diff
@@ -19,12 +63,63 @@ groups:
         note: >
           When the file name has multiple extensions (example.tar.gz), only the last one should
           be captured ("gz", not "tar.gz").
+      - id: fork_name
+        type: string
+        brief: >
+          Name of the fork. A fork is additional data associated with a filesystem object.
+        note: >
+          On Linux, a resource fork is used to store additional data with a filesystem object. A file always has at
+          least one fork for the data portion, and additional forks may exist.
+
+          On NTFS, this is analogous to an Alternate Data Stream (ADS), and the default data stream for a file is
+          just called $DATA. Zone.Identifier is commonly used by Windows to track contents downloaded from the Internet.
+          An ADS is typically of the form: C:\path\to\filename.extension:some_fork_name, and some_fork_name is the
+          value that should populate `fork_name`. `filename.extension` should populate `file.name`, and `extension`
+          should populate `file.extension`. The full path, `file.path`, will include the fork name.
+        stability: experimental
+        examples: ['Zone.Identifer']
+      - id: gid
+        type: string
+        brief: >
+          Primary Group ID (GID) of the file.
+        stability: experimental
+        examples: ["1000"]
+      - id: group
+        type: string
+        brief: >
+          Primary group name of the file.
+        stability: experimental
+        examples: ['users']
+      - id: inode
+        type: string
+        brief: >
+          Inode representing the file in the filesystem.
+        stability: experimental
+        examples: ['256383']
+      - id: mode
+        type: string
+        brief: >
+          Mode of the file in octal representation.
+        stability: experimental
+        examples: ['0640']
+      - id: mtime
+        type: string
+        brief: >
+          Time when the file content was last modified, in ISO 8601 format.
+        stability: experimental
+        examples: ['2021-01-01T12:00:00Z']
       - id: name
         type: string
         brief: >
           Name of the file including the extension, without the directory.
         stability: experimental
         examples: ['example.png']
+      - id: owner
```

Comment:
See the comment on `uid` - `file.owner.name|alias|username` would be more extensible